DfE.FindInformationAcademiesTrusts/Pages/Shared/_Layout.cshtml

```diff
@@ -32,6 +32,46 @@
 
   <partial name="_Footer"/>
   <script src="~/dist/main.js"></script>
+  <script type="text/javascript" src="~/dist/accessible-autocomplete/accessible-autocomplete.min.js"></script>
+  <script type="application/javascript">
```

Comment:
Can this be removed now that we are using the script on the search form partial?

Reply:
Right now this function is being called from the partial so we can't just bin it. Have made a task to look at moving it to a js file